pyclesperanto_prototype/_tier8/_deskew_y.py

```diff
@@ -35,5 +44,7 @@ def deskew_y(input_image: Image, output_image: Image = None, angle_in_degrees: f
     # correct orientation so that the new Z-plane goes proximal-distal from the objective.
     transform.rotate(angle_in_degrees=90, axis=0)
 
+    transform.scale_in_z_with_pixel_size(angle_in_degrees,pixel_size_xy=dxy,pixel_size_z=dz)
```

Comment:
Hi Pradeep @pr4deepr , just a quick question: instead of scaling with a sheared scaling factor, can we alternatively scale before the shearing? Can this achieve the same?

Reply:
Hi @haesleinhuepf  
   Is it just using the scale function?
BTW, I have been testing the RBC_test images and the deskewed images appear squashed when viewing them in napari. 
The dataset is from a Zeiss lattice where the  angle is 30 degrees. 
I'll get you the raw data and corresponding deskewed image from the microscope using proprietary software tomorrow so we can compare..

Cheers
Pradeep

Reply:
When thinking of an intuitive programming / user interface I think, we should ask the user to input the voxel size during acquisition AND the voxel size of the image that should be reconstrunted. That would be six numbers though...